web/client/components/share/ShareSocials.jsx

```diff
@@ -32,7 +32,14 @@ const {
 
 // icons of the social network
 const FacebookIcon = generateShareIcon('facebook');
-const TwitterIcon = generateShareIcon('twitter');
+const TwitterXIcon = ({ color = "#000000", size = 32, round = true }) => (
```

Comment:
why not using react-share?

Reply:
@MV88 did you see this https://github.com/geosolutions-it/MapStore2/issues/10983#issuecomment-2778730111 ?